keyboards/varanidae/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2023 QMK
+ * Copyright 2023 Silc Renew / Tokage IT Lab.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_BTN3,    KC_F16, KC_F17,  KC_F18,  KC_F19,    KC_F1,     KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9,   KC_F10, KC_F11, KC_F12,  KC_F13, KC_F14, KC_F15,
+        KC_F20,     KC_F21, KC_F22,  KC_F23,  KC_F24,    KC_ESC,    KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,    KC_9,   KC_0,   KC_MINS, KC_EQL, KC_DEL, KC_BSPC,
+        KC_PSLS,    KC_P7,  KC_P8,   KC_P9,   KC_HOME,   KC_TAB,      KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,   KC_LBRC,  KC_RBRC,  KC_ENT,
+        KC_PAST,    KC_P4,  KC_P5,   KC_P6,   KC_END,    KC_LALT,        KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,  KC_BSLS,
```

Comment:
The assignments of `KC_BSLS` and `KC_ENT` look wrong — for the ANSI layout the backslash key is above the Enter key, not below it.  If the wrong-looking code actually produced the expected mapping for the backslash and Enter keys, probably the layout definition needs to be corrected to match the actual PCB wiring in this area.
```suggestion
        KC_PSLS,    KC_P7,  KC_P8,   KC_P9,   KC_HOME,   KC_TAB,      KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,   KC_LBRC,  KC_RBRC,  KC_BSLS,
        KC_PAST,    KC_P4,  KC_P5,   KC_P6,   KC_END,    KC_LALT,        KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,  KC_ENT,
```

Reply:
I'm thinking of ISO Enter as the default layout for this keyboard, so KC_ENT is on top. are we forced to default to ANSI Enter?

Reply:
Ah, I understand. The problem is that K058 in LAYOUT is not in ISO shape; I specify "w": 1.25, "h": 2 for K058 to make it consistent. Is this correct?

Reply:
For now, I changed this to:
```
KC_LBRC,  KC_RBRC, KC_BSLS,
KC_SCLN, KC_QUOT,  KC_ENT,
```
for ansi,

and
```
KC_LBRC,  KC_RBRC,
KC_QUOT,  KC_NUBS, KC_ENT,
```

for iso and default.